aten/src/ATen/native/Convolution.cpp

```diff
@@ -338,7 +386,7 @@ at::Tensor _convolution(
   AT_CHECK(!params.is_padding_neg(), "negative padding is not supported");
```

Comment:
Can this two lines be moved into the `check_input_shape_forward` method?

Reply:
sure